### PR TITLE
IPFS integration

### PR DESCRIPTION
**Pull request by mikec**
adds IPFS for bounty metadata storage and retrieval

### CODE REVIEW

File: src/client/components/CreateBountyForm.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  }
+
+  const onClickHandler = async () => {
+    const { BugBounty, BugBountyFactory } = contracts()
+    const bugBountyFactory = await BugBountyFactory.deployed()
+
+    const createTx = await bugBountyFactory.createBugBounty(
+      500,
+      400,
+      300,
```
Comment (jpkcambridge) [inline src/client/components/CreateBountyForm.js]: What are these numbers?

Comment (mikec) [reply src/client/components/CreateBountyForm.js]: @jpkcambridge placeholders for payout amounts for high/critical/medium, etc

File: src/client/stores/CreateBountyFormStore.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { observable, action } from 'mobx'
+
+class CreateBountyFormStore {
```
Comment (jpkcambridge) [inline src/client/stores/CreateBountyFormStore.js]: Does this hold one bounty form, or many?  I feel like calling it a "store" implies many.

**Comment by jpkcambridge** [comment]
👍

File: src/client/stores/CreateBountyFormStore.js (hunk 2 above)
Comment (mikec) [reply src/client/stores/CreateBountyFormStore.js]: @jpkcambridge it holds the state for the form view

**Pull request merged**